onnx/defs/tensor/defs.cc

```diff
@@ -2538,164 +2538,178 @@ ONNX_OPERATOR_SET_SCHEMA(
-                          H_float_minus_one = Sub (H_float, one)
-                          W_float_minus_one = Sub (W_float, one)
-                          step_h_else = Div (two, H_float_minus_one)
-                          step_w_else = Div (two, W_float_minus_one)
-                          start_h_else = Identity (minus_one)
-                          start_w_else = Identity (minus_one)
-                      }
-                  >
-                  grid_w_steps_int = Range (int_zero, W, int_one)
-                  grid_w_steps_float = CastLike (grid_w_steps_int, step_w)
-                  grid_w_steps = Mul (grid_w_steps_float, step_w)
-                  grid_w_0 = Add (start_w, grid_w_steps)
-
-                  grid_h_steps_int = Range (int_zero, H, int_one)
-                  grid_h_steps_float = CastLike (grid_h_steps_int, step_h)
-                  grid_h_steps = Mul (grid_h_steps_float, step_h)
-                  grid_h_0 = Add (start_h, grid_h_steps)
-
-                  zeros_W_by_H = Transpose (zeros_H_by_W)
-                  grid_h_1 = Add (zeros_W_by_H, grid_h_0)
-                  grid_h = Transpose (grid_h_1)
-
-                  grid_w = Add (grid_w_0, zeros_H_by_W)
-
-                  # make following a function (theta, grid_w, grid_h) =>  (grid)
-                  original_grid_seq = SequenceConstruct (grid_w, grid_h, ones_H_by_W)
-                  original_grid = ConcatFromSequence <axis: int=-1, new_axis: int=1> (original_grid_seq)
-                  constant_shape_HW_3 = Constant <value_ints: ints = [-1, 3]> ()
-                  original_grid_HW_3 = Reshape (original_grid, constant_shape_HW_3)
-                  original_grid_3_HW_ = Transpose (original_grid_HW_3)
-
-                  original_grid_3_HW = CastLike (original_grid_3_HW_, theta)
-                  grid_N_2_HW = MatMul (theta, original_grid_3_HW)
-                  grid_N_HW_2 = Transpose <perm = [0, 2, 1]> (grid_N_2_HW)
-                  N_H_W_2_seq = SequenceConstruct (N, H, W, int_two_1d)
-                  N_H_W_2 = ConcatFromSequence <axis: int=-1, new_axis: int=0> (N_H_W_2_seq)
-                  grid_2d_then_ = Reshape (grid_N_HW_2, N_H_W_2)
-                  grid_2d_then = CastLike (grid_2d_then_, theta)
-                  },
-              else_branch = g2 () => (grid_3d_else) { # => (float[N, D, H, W, 3])
-                  int_one = Constant <value_int: int=1> ()
-                  minus_one = Constant <value = float {-1.0}> ()
-                  zero = Constant <value = float {0.0}> ()
-                  one = Constant <value = float {1.0}> ()
-                  two = Constant <value = float {2.0}> ()
+                  size_NCDHW_then = Concat <axis=0> (N, C, one_v, H, W)
+              },
+              else_branch = g2 () => (size_NCDHW_else) { # => (int64[5] = [N, C, D, H, W])
+                  # size_NCDHW_else = Identity (size)
+                  int_five = Constant <value_int: int=5> ()
```

Comment:
I don't think this `int_five` is used, right? Delete it?

Reply:
removed